src/app/app.component.ts

```diff
@@ -7,6 +7,6 @@ import { UserEntity } from './user-entity.model';
   styleUrls: ['./app.component.scss']
 })
 export class AppComponent {
-  userEntity: UserEntity = {id:1, firstName: 'Rahul', lastName: 'Patil'};
+  userEntity: UserEntity = {id: 1, firstName: 'Rahul', lastName: 'Patil'};
```

Comment:
1.create a constant  file and add data in that file
2. add I before interface IuserEntity

Reply:
commited above 2 changes

Reply:
create separate folder for interfaces and constants.

Reply:
Done above changes